examples/tfengine/devops.hcl

```diff
@@ -50,14 +50,21 @@ template "cicd" {
       owner = "GoogleCloudPlatform"
       name  = "example"
     }
-    branch_regex   = "^master$"
+    branch_name    = "master"
     terraform_root = "terraform"
 
+    # Required to create Cloud Scheduler jobs.
+    scheduler_region = "us-central1"
+
     # Prepare and enable default triggers.
     triggers = {
       validate = {}
-      plan     = {}
-      apply    = { run_on_push = false } # Do not auto run on push to branch
+      plan     = {
+        run_on_schedule = "0 12 * * *" # Run at 12 PM EST everyday
```

Comment:
Just so I understand, this will run on push AND run on schedule ?

Reply:
yea, run_on_push is default to true, so it doesn't need to be specified explicitly.